tests/test-mptcpwrap

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Test that libmptcpwrap injects MPTCP into the socket() system call.
+#
+# Copyright (c) 2021, Intel Corporation
+
+set -e
+
+LD_PRELOAD=../src/.libs/libmptcpwrap.so ./mptcpwrap-tester
```

Comment:
Do you think it's better to use `LD_PRELOAD` or to use a "normal" dynamic link?

Reply:
I was under the impression that the general use case was to preload libmptcpwrap.so so I did that for the test as well,, but I'm fine with "normal" dynamic link if you prefer.

Reply:
Given that our "normal" use case is LD_PRELOAD, probably best to stay close to that